catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/TableRepository.java

```diff
@@ -13,8 +13,8 @@
 
 package org.openmetadata.catalog.jdbi3;
 
+import static org.openmetadata.catalog.Entity.*;
```

Comment:
Should we avoid having wildcard imports in the codebase?

Reply:
We should, I don't how it was created.

Reply:
Check IntelliJ settings -- it does something automatically, if number of imports from the class exceeds a count.

I made it a large number (200) as shown below for "Class count to use import with * "

![image](https://user-images.githubusercontent.com/5906984/149829189-7b5d76ce-8129-4d29-b907-568ccc19055e.png)

Reply:
Mine is 999. No idea what happened.